Fix Makefile to make it more std compilant. The PREFIX variable works
now as expected and the SHAREDIR variable is added. So we can configure
the "make install" process to suit our needs.

diff -ruN wbar-1.3.3/Makefile wbar-1.3.3.new/Makefile
--- wbar-1.3.3/Makefile	2007-09-26 18:24:10.000000000 +0200
+++ wbar-1.3.3.new/Makefile	2009-02-16 15:46:37.000000000 +0100
@@ -1,8 +1,9 @@
 TARGET=wbar
 CXXFLAGS=`imlib2-config --cflags` -Wall -O2 #-DCOS_ZOOM #-DLINEAR_TRASL -DNO_EXPAND #-DAVGFILTER
 LDFLAGS=`imlib2-config --libs`  -Wl,-O2
-PREFIX=/usr/share/wbar
-
+PREFIX=/usr
+SHAREDIR=/share/wbar
+BINDIR=/bin
 	
 sources= XWin.cc Icon.cc Bar.cc IconLoader.cc \
 	 OptParser.cc SuperBar.cc SuperIcon.cc
@@ -18,30 +18,20 @@
 	#strip $(@)
 
 install: $(TARGET)
-	if [ "`whoami`" != "root" ]; then \
-		echo "You must be root to install"; \
-		exit 1; \
-	fi
-
-	install -d $(PREFIX)
-	awk '{if($$1 ~ /i:/ || ($$1 ~ /t:/ && NR<4)) print $$1" $(PREFIX)/"$$2; else print $$0;}' \
-		./dot.wbar > $(PREFIX)/dot.wbar
-	cp -a ./iconpack $(PREFIX)/iconpack
-
-	#ln -fs $(PREFIX)/iconpack/comic.ttf $(PREFIX)/iconpack/wbar.nuvoux/font.ttf
-	#ln -fs $(PREFIX)/iconpack/comic.ttf $(PREFIX)/iconpack/wbar.ice/font.ttf
-	ln -fs $(PREFIX)/iconpack/comic.ttf $(PREFIX)/iconpack/wbar.osx/font.ttf
+	install -d $(PREFIX)/$(SHAREDIR)
+	awk '{if($$1 ~ /i:/ || ($$1 ~ /t:/ && NR<4)) print $$1" $(PREFIX)/$(SHAREDIR)/"$$2; else print $$0;}' \
+		./dot.wbar > $(PREFIX)/$(SHAREDIR)/dot.wbar
+	cp -a ./iconpack $(PREFIX)/$(SHAREDIR)/iconpack
+
+	#ln -fs $(PREFIX)$(SHAREDIR)/iconpack/comic.ttf $(PREFIX)$(SHAREDIR)/iconpack/wbar.nuvoux/font.ttf
+	#ln -fs $(PREFIX)$(SHAREDIR)/iconpack/comic.ttf $(PREFIX)$(SHAREDIR)/iconpack/wbar.ice/font.ttf
+	ln -fs $(PREFIX)/$(SHAREDIR)/iconpack/comic.ttf $(PREFIX)/$(SHAREDIR)/iconpack/wbar.osx/font.ttf
 	
-	install ./wbar /usr/bin
+	install ./$(TARGET) $(PREFIX)$(BINDIR)
 
 uninstall:
-	if [ "`whoami`" != "root" ]; then \
-		echo "You must be root to install"; \
-		exit 1; \
-	fi
-
-	rm -rf $(PREFIX)
-	rm -f /usr/bin/wbar
+	rm -rf $(PREFIX)$(SHAREDIR)
+	rm -f  $(PREFIX)$(BINDIR)/$(TARGET)
 
 
 clean:
